backend/app/cmd/server.go

```diff
@@ -706,6 +734,7 @@ func (s *ServerCommand) makeAuthenticator(ds *service.DataStore, avas avatar.Sto
 			}
 			c.User.SetAdmin(ds.IsAdmin(c.Audience, c.User.ID))
 			c.User.SetBoolAttr("blocked", ds.IsBlocked(c.Audience, c.User.ID))
+			c.User.Email, _ = ds.GetUserDetail(store.Locator{SiteID: c.Audience}, c.User.ID, "email")
```

Comment:
Is updating email in the token here is enough? Or should we trigger this token renewal explicitly after setting an email?